websites/Y/YabancıDiziizlesene/presence.ts

```diff
@@ -5,34 +5,19 @@ const presence = new Presence({
     playing: "presence.playback.playing",
     paused: "presence.playback.paused",
     browsing: "presence.activity.browsing"
-  });
-
-/**
- * Get Timestamps
- * @param {Number} videoTime Current video time seconds
- * @param {Number} videoDuration Video duration seconds
- */
-function getTimestamps(
-  videoTime: number,
-  videoDuration: number
-): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
-  return [Math.floor(startTime / 1000), endTime];
-}
-
-const startTimestamp = Math.floor(Date.now() / 1000);
-
+  }),
+  startTimestamp = Math.floor(Date.now() / 1000);
+  
 let video: HTMLVideoElement;
 
-presence.on("iFrameData", async (msg) => {
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
```

Comment:
You can't assign this type to this message lol, you are not sending video element, you are sending duration, currenttime and paused state, so do custom interface for it... and assign it.